apps/mobile/src/screens/onboarding/onboarding.tsx

```diff
@@ -46,26 +46,25 @@ const Onboarding: React.FC = () => {
 	const navigation =
 		useNavigation<NativeStackNavigationProp<RootStackParameterList>>();
 
+	const allAnswers = useAppSelector(
+		(state) => state.onboarding.answersByQuestionIndex,
+	);
+	const currentAnswer = useAppSelector(
+		(state) => state.onboarding.answersByQuestionIndex,
+	);
```

Comment:
Let's combine all `useAppSelector` calls

Reply:
@fshabanov, need your advice here. 
I saw this warning after merge from main and found that issue is because of the object returned from the `useAppSelector` in the `OnboardingScreen`. That's why I broke it to primitives and for each of them called useAppSelector.
> Selector unknown returned a different result when called with the same parameters. This can lead to unnecessary rerenders.
> Selectors that return a new reference (such as an object or an array) should be memoized: https://redux.js.org/usage/deriving-data-selectors#optimizing-selectors-with-memoization

I found next in :

Reply:
We're okay with this warning for now, we don't want to have different code styles throughout the codebase

Reply:
I guess for now we can use `LogBox.ignoreLogs()` just not to show this warning